test/_helpers.js

```diff
@@ -37,19 +39,20 @@ export function compile (entry, configDecorator) {
       output: {
         path: path.resolve(__dirname, path.resolve(context, 'dist')),
         filename: 'bundle.js',
-        chunkFilename: '[name].chunk.js'
+        chunkFilename: '[name].chunk.js',
       },
       resolveLoader: {
-        modules: [path.resolve(__dirname, '../node_modules')]
+        modules: [path.resolve(__dirname, '../node_modules')],
       },
       module: {
-        rules: []
+        rules: [],
       },
-      plugins: []
+      plugins: [],
```

Comment:
I'm happy to have Prettier in this repo, but I'd prefer to have dangling commas disabled if you're alright with it.